islandora_openseadragon.install

```diff
@@ -35,6 +51,46 @@ function islandora_openseadragon_uninstall() {
     'islandora_openseadragon_tile_size',
     'islandora_openseadragon_tile_overlap',
     'islandora_openseadragon_fit_to_aspect_ratio',
+    'islandora_openseadragon_tilesource',
+    'islandora_openseadragon_djatoka_url',
+    'islandora_openseadragon_iiif_url',
+
+  );
+  array_walk($variables, 'variable_del');
+}
+
+/**
+ * Update settings to new format.
+ *
+ * Migrates any options from the old version to the new version if they still
+ * exist.
+ */
+function islandora_openseadragon_update_7100(array &$sandbox) {
+  module_load_include('inc', 'islandora_openseadragon', 'includes/admin.form');
+  module_load_include('inc', 'islandora_openseadragon', 'includes/utilities');
+  // Get new OpenSeadragon default settings.
+  $new_settings = islandora_openseadragon_default_options();
+  // Get the old settings.
+  $old_settings = variable_get('islandora_openseadragon_settings', array());
+
+  if (isset($old_settings['djatokaServerBaseURL'])) {
```

Comment:
Or we can just keep the old name of it, that's probably for the best even if it looks inconsistent.